onnxmltools/utils/metadata_props.py

```diff
@@ -0,0 +1,67 @@
+from ..proto import onnx, onnx_proto
+from distutils.version import StrictVersion
+
+
+def _get_case_insensitive(iterable, key):
+    return next((x for x in iterable if x.lower() == key.lower()), None)
+
+
+def _validate_metadata(metadata_props):
```

Comment:
Please add `# metadata_props is a Python dictionary. For each key-value pair, the key and value specify a variable's name and its actual value, respectively.` This might help potential devs to know your work more.